### PR TITLE
Update train.py

### PR DESCRIPTION
**Pull request by Hima-Mehta**
Made necessary changes according to Tf release 1.0 to make it compatible with tensorflow 1.0.

### CODE REVIEW

**Comment by Hima-Mehta** [comment]
Sorry for Confusion. I was trying to get it reviewed. Can you please review the changes ? I would like to use this proposed ANN further.

**Comment by jostmey** [comment]
I will take a look at this as soon as I can. My wife is in labor... so it could be a few days :-(

**Comment by Hima-Mehta** [comment]
Oh congrats and good luck! :)

**Comment by jostmey** [comment]
So there are a few issues with the changes. The actual upgrade from Tensorflow 0.12 to Tensorflow 1.0 looks valid. But the code changes also include using a different loss function. The MSE has been replaced by a binary cross entropy error. For the adding problem, the MSE error is the what I reported in the manuscript, so I need to keep it the same. For other types of problems, the binary cross entropy would be more appropriate.

I will try to accept your changes today, and then fix the code changes

**Comment by axiomying** [comment]
Good
